pkgs/applications/video/giph/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, ffmpeg
+, xdotool
+, withSlop ? true
+, slop
+, withNotify ? true
+, libnotify
+, withPgrep ? true
+, procps
+, makeWrapper
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "giph";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "phisch";
+    repo = "giph";
+    rev = version;
+    sha256 = "19l46m1f32b3bagzrhaqsfnl5n3wbrmg3sdy6fdss4y1yf6nqayk";
+  };
+
+  dontConfigure = true;
+
+  dontBuild = true;
+
+  installFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/giph --prefix PATH : ${lib.makeBinPath ([ ffmpeg xdotool ] ++
+      lib.lists.optional withNotify libnotify ++
+      lib.lists.optional withSlop slop ++
+      lib.lists.optional withPgrep procps
+    )}
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/phisch/giph";
+    description = "Simple gif recorder";
+    license = licenses.mit;
+    maintainers = [ maintainers.legendofmiracles ];
```

```suggestion
    maintainers = [ maintainers.legendofmiracles ];
    platforms = platforms.linux;
```
Since the program depends on `xdotool`, I assume it will only work on Linux.

---

Good call, completly forgot that